Assets/Scripts/Models/JobQueue.cs

```diff
@@ -35,6 +35,7 @@ public int GetCount()
 
     public void Enqueue(Job j)
     {
+        Debug.Log("Enqued Job" + (j.tile == null ? "." : (" at (" + j.tile.X + ", " + j.tile.Y + ").")));
```

Comment:
Better to use `Debug.ULogChannel` than regular `Debug.Log` particularity with something like this that is potentially quite spammy.
And it is spelled Enqueued.